acct.bionimbus.org/etlMapping.yaml

```diff
@@ -126,7 +126,7 @@ mappings:
         fn: count
     joining_props:
       - index: file
-        join_on: _subject_id
+        join_on: subject_id
```

Comment:
Why did this row change?

Reply:
The automation is replicating whatever it finds in `gitops-qa/qa-acct.planx-pla.net`.
If ETL is working fine in QA, then I'd expect the same to work fine in PROD.
cc: @mpingram also holds ETL knowledge so perhaps we can help us +1 that.